test/terminal_casl2comet2_test.py

```diff
@@ -0,0 +1,264 @@
+        e.clear()
+        buf_to_js = buf.replace('\\', '\\\\').replace('\n', '\\n').replace('\"', '\\"')
+        driver.execute_script(f'document.getElementById("casl2src").value="%s"' % buf_to_js)
+        # e.send_keys(buf)
+        e = driver.find_element(By.ID, "assemble")
+        e.click()
+        e = driver.find_element(By.ID, "terminal-2")
+        if not "DEFINED SYMBOLS" in e.text:
+            raise Casl2AssembleError
+        e = driver.find_element(By.ID, "terminal-1")
+        ActionChains(driver)\
+            .move_to_element(e)\
+            .click()\
+            .send_keys("run")\
+            .send_keys(Keys.RETURN)\
+            .perform()
+        with open("input.json") as fp:
+            input = json.load(fp)
+        if os.path.basename(casl2_file) in input.keys():
+            for in_str in input[os.path.basename(casl2_file)]:
+                ActionChains(driver)\
+                    .move_to_element(e)\
+                    .click()\
+                    .send_keys(in_str)\
+                    .send_keys(Keys.RETURN)\
+                    .perform()
+        WebDriverWait(driver, timeout).until(expected_conditions.text_to_be_present_in_element((By.ID, "terminal-1"), "[Program finished]"))
+        terminal_text = e.text
+        with open(out_file, mode='w') as fp:
+            out_match = re.search(r"^run\n(.*)\[Program finished\]", terminal_text, flags=re.MULTILINE | re.DOTALL)
+            if out_match:
+                fp.write(out_match.group(1))
+    except TimeoutException:
+        terminal_text = e.text
+        with open(out_file, mode='w') as fp:
+            fp.write("============TIMEOUT==============\n")
+            out_match = re.search(r"^run\n(.*)", terminal_text, flags=re.MULTILINE | re.DOTALL)
+            if out_match:
+                fp.write(out_match.group(1))
+        raise TimeoutException(f"Timeout ({timeout}sec)")
+    except Casl2AssembleError:
+        with open(out_file, mode='w') as fp:
+            fp.write("============ASSEMBLE ERROR==============\n")
+            fp.write(e.text)
+        raise Casl2AssembleError
+    except Exception as err:
+        with open(out_file, mode='w') as fp:
+            print(err, file=fp)
+        raise err
+
+
+if __name__ == "__main__":
+    sample_files = glob.glob("samples/**/*.cas", recursive=True)
+    if len(sys.argv) != 2:
+        print("Please specify the browser to use in the argument.(firefox|chrome)")
+        sys.exit(-1)
+    if sys.argv[1] == "firefox":
+        driver = init_firefox_driver()
+    elif sys.argv[1] == "chrome":
+        driver = init_chrome_driver()
+    else:
+        print("You can only specify firefox or chrome for your browser.")
+        sys.exit(-1)
+    passed = 0
+    failed = 0
+    if not os.path.exists("test_result"):
+        os.mkdir("test_result")
+    try:
+        # for src in sample_files:
+        src = "samples/program1/sample16.cas"
+        if src:
```

Comment:
コメントアウトの戻し忘れかな
```suggestion
        for src in sample_files:
```

Reply:
```suggestion
        for src in sample_files:
        # src = "samples/program1/sample16.cas"
        # if src:
```
すみません、これ上記が正しいです...
sample16.casのタイムアウトを何度も確かめてたときの名残です...
コメントアウト部は消しても構いません